Mage/src/main/java/mage/abilities/SpellAbility.java

```diff
@@ -306,7 +306,7 @@ public Card getCharacteristics(Game game) {
     }
 
     public static SpellAbility getSpellAbilityFromEvent(GameEvent event, Game game) {
-        if (event.getType() != GameEvent.EventType.CAST_SPELL) {
+        if (event.getType() != GameEvent.EventType.CAST_SPELL && event.getType() != GameEvent.EventType.CAST_SPELL_LATE) {
```

Comment:
can you explain this change?

Reply:
Getting the name of a spell for comparison happens in CAST_SPELL_LATE. I'm not sure what this function used to be used for (it currently had no usage) but since I needed basically this function and it was already in the code, I used it with the small change.